relay-system/Cargo.toml

```diff
@@ -13,4 +13,6 @@ publish = false
 actix = "0.7.9"
 failure = "0.1.8"
 futures = "0.1.28"
+futures03 = { version = "0.3", package = "futures", features = ["compat"] }
 relay-log = { path = "../relay-log" }
+tokio = { version = "1.0", features = ["rt-multi-thread"] } # in sync with reqwest
```

Comment:
What does the comment here express?

Reply:
Was an Artifact from copying the line from another toml file